packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -9,7 +9,6 @@ import { useRouter } from 'next/router';
 import { Spaciness } from '../../graphql/settings';
 import SettingsContext from '../../contexts/SettingsContext';
 import FeedContext from '../../contexts/FeedContext';
-import ScrollToTopButton from '../ScrollToTopButton';
```

Comment:
This was previously removed in https://github.com/dailydotdev/apps/pull/2553/files, but has been brougt back on search merge PR, I believe by mistake.

Reply:
Probably, there were lot of conflicts there so git probably creeped it back in. 🙏